ui/components/app/incoming-trasaction-toggle/incoming-transaction-toggle.component.js

```diff
@@ -0,0 +1,84 @@
+import React, { useContext } from 'react';
```

Comment:
Can we remove the .component extension from this file and just rename it to ` incoming-transaction-toggle.js`.  A while ago we decided to avoid using this nomenclature for new components cc @darkwing

Reply:
Good idea, much simpler